Add fix for apparently missing explicit dependency on libX11, as a result of which compilation fails (X_LIBS in configure.ac not being correctly set);
this may need further investigation.

Signed-off-by: David McIlwraith <archaios@archaios.net>
---
 configure.ac |    3 +++
 1 file changed, 3 insertions(+)

Index: configure.ac
===================================================================
--- configure.ac        (revision 18215)
+++ configure.ac        (working copy)
@@ -404,6 +404,9 @@
 # Put this here so we get the gdk libs
 # AC_CHECK_LIB(gdk_imlib, main,GTK_LIBS="$GTK_LIBS -lgdk_imlib")
 
+# Added due to link errors that occur sporadically due to the absence of an explicit -lX11 during compilation
+AC_CHECK_LIB(X11, main, X_LIBS="$X_LIBS -lX11")
+
 # Need to check for Xext while we still have X_LIBS set.
 AC_CHECK_LIB(Xext, main, X_LIBS="$X_LIBS -lXext")

